esi_leap/common/keystone.py

```diff
@@ -73,3 +73,11 @@ def get_project_name(project_id, project_list=None):
         return project.name if project else ''
     else:
         return ''
+
+
+def get_project_email(project_id):
+    if project_id:
+        project = get_keystone_client().projects.get(project_id)
+        return project.email if project else ''
```

Comment:
What if a project doesn't have an email attribute? Will this still work?

Reply:
Oh, yes. I changed it to getattr(project, 'email', None) to get email value or None